torch/distributed/pipelining/schedules.py

```diff
@@ -94,6 +96,8 @@ def from_str(action):
             return _ComputationType.SEND_B
         elif action == "RECV_B":
             return _ComputationType.RECV_B
+        elif action == "BW":
```

Comment:
For weight-deferred schedules (e.g. interleaved1f1b), `B` stands for a "full backward", while in zero-bubble `B` stands for "backward for inputs" and `BW` actually stands for the full backward. So maybe for better consistency actually the non-zerobubble schedules should be using `BW` in their schedules instead of `B`? Or should we just give a note about this to the users.

Reply:
Yes, if you are good with the current way (BW=FULL_BACKWARD) I can update all the other schedules in this PR.  I also need to add the execution part as you pointed out.

Reply:
Alternatively we could keep B=FULL_BACKWARD and add a new I=INPUT for input only? I'm fine with either